### DISCUSSION TITLE
Coding

### DISCUSSION THREAD

**Discussion by Maderiahmed** [community/community#38648] [Other Feature Feedback, Questions, & Ideas] [answered]
How to learn jquery ?

**Comment by DSDmark** [comment, marked as answer]
Hi @Maderiahmed,

There are lot of options for learn **jQuery** like:-

- jQuery Learning Center (it's an official documentation) :- [**jQuery learning center**](https://learn.jquery.com/)
- w3school :- [**w3school**](https://www.w3schools.com/jquery/default.asp)
- javapoint :- [**javapoint**](https://www.javatpoint.com/jquery-tutorial)
- tutorialspoint :- [**tutorialspoint**](https://www.tutorialspoint.com/jquery/index.htm)
- and so no.... 

I suggest w3school or jQuery learning center.
